src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -542,7 +542,7 @@ void SizeComputeCPU(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(req.size(), 1U);
   const TBlob& in_data = inputs[0];
   const TBlob& out_data = outputs[0];
-  const index_t size_var = in_data.Size();
+  const int64_t size_var = in_data.Size();
   memcpy(out_data.dptr_, &size_var, 1U * sizeof(int64_t));
```

Comment:
No need to do `1U *` either.

Reply:
If I would want to assign a int64 variable a value, I would rather go for: 
```*static_cast<int64_t*>(out_data.dptr_) = in_data.Size();```

Reply:
Yeah I agree, better to cast the pointer and then assign.